recipes/php.rb

```diff
@@ -34,3 +34,8 @@
   user node['nginx']['user']
   group node['nginx']['group']
 end
+
+execute 'run laravel migrations to setup database' do
```

Comment:
Not familiar with php artisan, but is there a particular reason we need to use the `--force` flag? Without it, could the resource possibly be idempotent?

Reply:
`php artisan migrate` is intended to be run interactively, and will no-op if there is nothing to migrate.

This is currently a workaround for #5.